### ISSUE TITLE
Enabled "python.analysis.autoFormatStrings" is being applied on docstrings

### ISSUE THREAD

**Issue by alfawal** [microsoft/pylance-release#4583]
## Environment data

-   Language Server version: v2023.7.10
-   OS and version: MacOS 13.3.1
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.11.3

## Code Snippet

```python
def convert_values_to_two_decimal_places_string(dict_: dict[Any, float]) -> dict[Any, str]:
    """Converts the values from floats to strings with two decimal places.

    Args:
        dict_ (dict[Any, float]): The dictionary to convert its values.

    Returns:
        dict[Any, str]: The dictionary with its values converted to a string with two decimal places.

    Usage:
    >>> convert_values_to_two_decimal_places_string({"pi": 3.14159265359})
    {"pi": "3.14"}
    """
    return {key: f"{value:.2f}" for key, value in dict_.items()}
```

## Repro Steps

1. Create a docstring.
2. Add {} anywhere inside of it.

## Expected behavior

Pylance not considering it as an f-string.
```py
"""
{}
"""
```

## Actual behavior

Pylance considers it as an f-string and adds leading f to convert it to an f-string.
```py
f"""
{}
"""
```

## Logs

Not important in this case.

**Comment by rchiodo** [comment]
Thanks for the bug. It should be changed to skip this in doc strings.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.7.21, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023721-12-July-2023-PreRelease)
